src/service/feedback.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2024 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { Dialog, Notification } from '@jupyterlab/apputils';
+import { ReactWidget } from '@jupyterlab/ui-components';
+
+import { postFeedback } from './api';
+import { getCurrentModel } from './modelHandler';
+import { lastPrompt } from '../QiskitCompletionProvider';
+import { feedbackIcon } from '../utils/icons';
+import { IFeedbackForm, IFeedbackResponse } from '../utils/schema';
+
+const toBool = (bool?: string): boolean | undefined => {
+  switch (bool) {
```

Comment:
not a big deal now but would it be better/future proof if the `bool` string is set to lowercase

Reply:
perhaps this helper function's name is deceptive, this actually converts the value of a radio button defined on line 122/127 to a boolean. The values `"true"` and `"false"` are set on those lines and could actually be any string we want, I could have had them be `"thumbsup"` and `"thumbsdown"`, but I figured truthy values would be easier.

so no, the `"true"` and `"false"` here is a enum not a string representation of a boolean, and so any other values would be considered "wrong"